### ISSUE TITLE
[Issue]: Wii Classic Controller Pro Gamepad mode bumper and triggers are opposite

### ISSUE THREAD

**Issue by JMartinez9820** [sanjay900/guitar-configurator#3]
### Please describe your issue.

The L and R bumpers, and ZL and ZR trigger buttons on the Wii Classic Controller Pro flipped around. L and R correspond to LT and RT, while ZL and ZR are correspond to LB and RB. It should be the other way around where L and R correspond to LB and RB, while ZR and ZR correspond to LT and RT. Although some games allow you to remap certain buttons, it can get slightly annoying when you think you're pressing L or R (LB or RB), but in reality you're pressing LT and RT. I hope this gets fixed in an update because I just played a bit of TMNT Shredder's Revenge with my SNES Classic Edition Controller, and I can't do the ability to find friends. It seems that A and B are also flipped in Xinput? X correctly corresponds to Y and X correctly corresponds to Y, but A and B are the same. B should be A, and A should be B on XInput.

### Version

yes

### What operating system are you using?

Windows 11

### What microcontroller are you using?

pi pico

### How is your controller programmed?

Wii

**Comment by JMartinez9820** [comment]
Another bug I found:
The X and Y buttons are reversed and correspond to the wrong buttons. X should correspond to X, Y should correspond to Y.

**Comment by sanjay900** [comment]
Yeah, the flipped face buttons is something that will be fixed by the new
firmware as well. It's one of those things where depending on the scenario
both layouts make sense, but the new firmware gives you the option for both

**Comment by sanjay900** [comment]
Zl and zr I have probably also fixed but I'll confirm that.

**Comment by sanjay900** [comment]
But you can also do your own bindings either way with the new firmware
